pages/1.11/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -230,7 +230,7 @@ In this case the custom CA certificate is an intermediate one, issued directly b
 - on the bootstrap node:
     - `$DCOS_INSTALL_DIR/genconf/dcos-ca-certificate.crt` file containing the custom CA certificate in PEM format
     - `$DCOS_INSTALL_DIR/genconf/dcos-ca-certificate-key.key` file containing the private key associated with the custom CA certificate in the PKCS#8 format
-    - `$DCOS_INSTALL_DIR/genconf/dcos-ca-certificate-chain.crt` file containing the certificate chain in PEM format
+    - `$DCOS_INSTALL_DIR/genconf/dcos-ca-certificate-chain.crt` file containing only the root CA certificate in PEM format
```

Comment:
Is this true? Does the `chain` file contain only the root CA certificate?

Reply:
Use case 2 is specifically about the case with a single intermediate CA. You can see this fact already on line 228 above. It might help to look at where this occurs on the current page: https://docs.mesosphere.com/1.13/security/ent/tls-ssl/ca-custom/#use-case-2

 The change is to repeat it explicitly here, since it was described as confusing during a COPS issue.